videointelligence/google/cloud/videointelligence_v1p2beta1/gapic/video_intelligence_service_client.py

```diff
@@ -38,8 +39,9 @@
 from google.cloud.videointelligence_v1p2beta1.proto import video_intelligence_pb2_grpc
 from google.longrunning import operations_pb2
 
+
 _GAPIC_LIBRARY_VERSION = pkg_resources.get_distribution(
-    "google-cloud-videointelligence"
+    "google-cloud-video-intelligence"
```

Comment:
Not sure what generator change is making this happen. :confused:

Reply:
Looks like we need to extend this synth replacement to cover the spread out version: https://github.com/googleapis/google-cloud-python/blob/ba1ddc2d54dcfa395be45dee9dd4cb35dd49c138/videointelligence/synth.py#L52-L56

Reply:
@busunkim96 I have a manual synth PR coming which fixes the replacement.